webui/react/src/pages/Templates/TemplateDeleteModal.tsx

```diff
@@ -0,0 +1,43 @@
+import { Modal } from 'hew/Modal';
+
+import { deleteTaskTemplate } from 'services/api';
+import { Template } from 'types';
+import handleError, { ErrorLevel, ErrorType } from 'utils/error';
+
+interface Props {
+  template: Template;
+  onSuccess?: () => void;
```

Comment:
**nit suggestion:** this might be better as `onComplete` since it's called even when deletion fails

Reply:
Actually I believe this function should only trigger when deletion completes.